extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -586,6 +572,106 @@ export class AmpConsent extends AMP.BaseElement {
     this.policyConfig_ = config['policy'] || this.policyConfig_;
   }
 
+  /**
+   * Read the inline config from publisher
+   * @return {?JsonObject|undefined}
+   */
+  getInlineConfig_() {
+    // All consent config within the amp-consent component. There will be only
+    // one single amp-consent allowed in page.
+    // TODO: Make this a shared helper method.
```

Comment:
can we do it now?

Reply:
I tried adding the helper method. It's more complicated than I originally thought because
1. Error messages are different for different extensions. 
2. The mix between user().assert and user().error.
3. Minor differences like whether can there be extra child besides `<script>`. Can there be multiple `<script>` ...

Reply:
how about having a shared function like:

```
function getSingleChildJsonConfig(element);
```

It prints standardized `user().error()` and returns `null` if the config is invalid. The caller decides whether to throw. 

You don't need to refactor all the other extensions now. But moving it to a shared function will be the 1st step for such bigger refactoring.